src/app/containers/AudioVideo/Canonical.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styled from 'styled-components';
+import { string, bool } from 'prop-types';
+
+const Canonical = ({ id, portrait }) => {
+  const position = portrait ? 'position: absolute;' : '';
+
+  const PortraitParent = styled.div`
+    padding-bottom: 178.78%;
```

Comment:
Can we document anywhere (maybe a comment) why `178.78%` is being set here?

Reply:
To clarify, this is how we represent a 9:16 ratio.

16:9 = `(100/16)*9` = `56.25%`
9:16 = `(100/9)*16` = `177.78%`

Reply:
Will do as part of https://github.com/bbc/simorgh/issues/2412